### PR TITLE
Fixing a typo in Equation 20 of kim21h.pdf and adding the middle initial of the last author

### PR DESCRIPTION
**Pull request by timkimd**
I corrected a typo in the PDF by changing $f_\theta$ to $f_\psi$, and added the middle initial of the last author.

### CODE REVIEW

**Comment by timkimd** [comment]
No other edits other than the correction of typos were made to the PDF.

**Comment by lawrennd** [comment]
Hi Tim, Thanks for the PR! 

I'm not sure of the final process, but I think ICML are/were going to ask for a round of updates that their proceedings editor will integrate (via pull requesrs). Has that call gone out yet?

**Comment by timkimd** [comment]
Hi Neil, thank you for the reply! I don't think I received a notice asking authors to make pull requests yet. If it were possible to make further edits, I hope I could add a couple more sentences and crucial references, based on feedback that I got during ICML.

**Comment by lawrennd** [comment]
I *think* that’s the plan but I may be confusing conferences (we just did COLT as well).

In any case, for a PDF change, I look for some sort of confirmation from the Program Chairs or Publication chair ... just because different conferences have different standards. When it's just the meta-information I'll normally merge those on my own .. because I can normally check against the PDF.

File: _posts/2021-07-01-kim21h.md [hunk 1]
```diff
@@ -32,15 +32,15 @@ page: 5551-5561
 order: 5551
 cycles: false
 bibtex_author: Kim, Timothy D and Luo, Thomas Z and Pillow, Jonathan W and Brody,
-  Carlos
+  Carlos D
```
Comment (lawrennd) [inline _posts/2021-07-01-kim21h.md]:
```suggestion
  Carlos D.
```

File: _posts/2021-07-01-kim21h.md [hunk 2]
```diff
@@ -32,15 +32,15 @@ page: 5551-5561
 order: 5551
 cycles: false
 bibtex_author: Kim, Timothy D and Luo, Thomas Z and Pillow, Jonathan W and Brody,
```
Comment (lawrennd) [inline _posts/2021-07-01-kim21h.md]:
```suggestion
bibtex_author: Kim, Timothy D. and Luo, Thomas Z. and Pillow, Jonathan W. and Brody,
```

File: _posts/2021-07-01-kim21h.md [hunk 3]
```diff
@@ -32,15 +32,15 @@ page: 5551-5561
...
-  Carlos
+  Carlos D
 author:
 - given: Timothy D
   family: Kim
 - given: Thomas Z
   family: Luo
 - given: Jonathan W
```
Comment (lawrennd) [inline _posts/2021-07-01-kim21h.md]:
```suggestion
- given: Jonathan W.
```

File: _posts/2021-07-01-kim21h.md [hunk 4]
```diff
@@ -32,15 +32,15 @@ page: 5551-5561
 order: 5551
 cycles: false
 bibtex_author: Kim, Timothy D and Luo, Thomas Z and Pillow, Jonathan W and Brody,
-  Carlos
+  Carlos D
 author:
 - given: Timothy D
```
Comment (lawrennd) [inline _posts/2021-07-01-kim21h.md]:
```suggestion
- given: Timothy D.
```

File: _posts/2021-07-01-kim21h.md [hunk 5]
```diff
@@ -32,15 +32,15 @@ page: 5551-5561
...
   family: Kim
 - given: Thomas Z
```
Comment (lawrennd) [inline _posts/2021-07-01-kim21h.md]:
```suggestion
- given: Thomas Z.
```

File: _posts/2021-07-01-kim21h.md [hunk 6]
```diff
@@ -32,15 +32,15 @@ page: 5551-5561
...
   family: Luo
 - given: Jonathan W
   family: Pillow
-- given: Carlos
+- given: Carlos D
```
Comment (lawrennd) [inline _posts/2021-07-01-kim21h.md]:
```suggestion
- given: Carlos D.
```

**Review by lawrennd — APPROVED** [review]
Made some minor fixes (all initials need a period after them for latex to understand they are initials).

**Pull request merged**